spec/support/spec_config.rb

```diff
@@ -526,6 +530,9 @@ def root_user
 
   # Get the default test user for the suite on versions 2.6 and higher.
   def test_user
+    # When testing against a serverless instance, we are not allowed to create
+    # new users, we just have one user for everyhing.
+    return root_user if Utils.serverless?
```

Comment:
Can you please add a blank line after this one to separate this piece from the rest of the method.

Reply:
Done